Sources/WordPressData/Resources/WordPress.xcdatamodeld/WordPress 153.xcdatamodel/contents

```diff
@@ -814,7 +814,7 @@
         <attribute name="slug" attributeType="String" syncable="YES"/>
         <relationship name="blog" maxCount="1" deletionRule="Nullify" destinationEntity="Blog" inverseName="roles" inverseEntity="Blog" syncable="YES"/>
     </entity>
-    <entity name="SharingButton" representedClassName="WordPress.SharingButton" syncable="YES">
+    <entity name="SharingButton" representedClassName="SharingButton" syncable="YES">
```

Comment:
There is no `.` in the `SharingButton` class name, and a few other entities. Is that expected?